lib/iris/tests/results/imagerepo.json

```diff
@@ -1,952 +1,982 @@
 {
     "example_tests.test_COP_1d_plot.TestCOP1DPlot.test_COP_1d_plot.0": [
-        "https://scitools.github.io/test-iris-imagehash/images/v4/baff589936602d8ec977334ae4dac9b61a6dc4d99532c86cc2913e36c4cc0f61.png",
+        "https://scitools.github.io/test-iris-imagehash/images/v4/baff589936602d8ec977334ae4dac9b61a6dc4d99532c86cc2913e36c4cc0f61.png", 
```

Comment:
This must be an editor thing. My PR removed all of the extra whitespace at the end of each line and yours adds it back in!

Reply:
But I didn't use an editor at all.
It seems to be just what happens when you run tests with "IRIS_TEST_CREATE_MISSING=1" ?